### ISSUE TITLE
Yoyo loader crash after trying to download an animated banner

### ISSUE THREAD

**Issue by SofyUchiha** [Rinnegatamante/yoyoloader_vita#30]
After trying numerous times to download animated banners by pressing select from yoyo loader, the app crashes.
It throws error C2-12828-1
Yoyo loader: v. 1c5f2dc

**Comment by Rinnegatamante** [comment]
Pushed a fix, will be available on nightly in few mins.

**Comment by vin50cc** [comment]
Hello Rinnegatamante, Sorry but I do not know where else to turn.  It concerns the installation of "YoYo Launcher" on mu modded PS Vita.  I have spent the last several days (after watching youtuber "Zealous Chuck") to install said app while following both yours and Github instructions.  I have latest Nightly Version.  I also have installed thru VPK "libshacccg.suprx" with success.  I also installed a few games (i.e. AlienEscape.apk, LM2D.apk, CC4.apk) in ux0:data/gms folder.  I understand that I should NOT install "fd_fix.skprx" if RePatch Plugin is installed (such is my case, so I did NOT install).  Based on the aforementioned, I am starting to suspect that perhaps my ux0 and ur0 files are in conflict???  I would be willing to send you any attachment of said files if necessary.  Any comment is most greatly appreciated and I thank you in advance for your valuable time & assistance!!
